packages/oruga/src/composables/useInputHandler.ts

```diff
@@ -35,6 +36,17 @@ function asValidatableFormElement(el: unknown): ValidatableFormElement | null {
         : null;
 }
 
+const constraintValidationAttributes = [
+    "disabled",
+    "required",
+    "pattern",
+    "maxlength",
+    "minlength",
+    "max",
+    "min",
+    "step",
+];
```

Comment:
I can do that, but [this line](https://github.com/oruga-ui/oruga/pull/930/files/126763ed1fb2e3d96c0605ea53b54a68b9993d0c#diff-e8e2be411f6a4c43f5676dd3bffe86bda2ad8c1420749d6376f37f7b8a75566dR276) then starts complaining because the `as const` implicitly makes the array `readonly`. TypeScript's built-in DOM types don't mark `attributeFilter` as `readonly`, so the object construction is marked as an error. Based on the DOM API specification, the parameter is a WebIDL `sequence`, which is passed by copy and not mutated in-place, so TypeScript _could_ safely mark that part of the call signature as `readonly`, but it hasn't done so yet.